src/pages/penjualan.tsx

```diff
@@ -1,8 +1,21 @@
+import { AdminLayout } from '@layouts/AdminLayout'
+import { KaryawanLayout } from '@layouts/KaryawanLayout'
+import { useMarContext } from 'src/context/MarFashionProvider'
 import { DataPenjualan } from '../containers/Penjualan'
 
 const NotaPenjualanPage = () => {
+  const { user } = useMarContext()
   return (
-    <DataPenjualan />
+    <div>
+      { user?.role == "ADMIN" ?
+        <AdminLayout>
+          <DataPenjualan />
+        </AdminLayout> :
+        <KaryawanLayout>
+          <DataPenjualan />
+        </KaryawanLayout>
```

Comment:
you can create another Layout, so you dont have to create condition for every page.